Dockerfile

```diff
@@ -27,31 +27,39 @@ RUN sed -i '1ideb mirror://mirrors.ubuntu.com/mirrors.txt bionic main restricted
     sed -i '1ideb mirror://mirrors.ubuntu.com/mirrors.txt bionic-updates main restricted' /etc/apt/sources.list && \
     sed -i '1ideb mirror://mirrors.ubuntu.com/mirrors.txt bionic-security main restricted' /etc/apt/sources.list
 
+
+
 # Install build dependencies
 RUN dpkg --add-architecture i386 && apt-get update &&                       \
-    apt-get -y install build-essential cmake wget texinfo flex bison        \
+    apt-get -y install build-essential wget texinfo flex bison        \
     python-dev python3-dev python3-venv python3-distro mingw-w64            \
-    lsb-release libgomp1 unzip
+    lsb-release libgomp1 unzip libedit-dev libxml2-dev
+
+# We need a new-ish cmake to deal with LLVM-13 binary packages since the
+# default on 18.0 is too old. Use latest as of this writing.
```

Comment:
This is a good comment. It will help a lot when upgrading the docker file to a more recent distribution.